spectractor/extractor/spectroscopy.py

```diff
@@ -493,10 +489,7 @@ def print_detected_lines(self, output_file_name="", overwrite=False, print_table
             for col in t.colnames[-2:]:
                 t[col].unit = 'nm'
             t[t.colnames[-3]].unit = 'reduced'
-            if output_file_name != "":
-                t.write(output_file_name, overwrite=overwrite)
-            if print_table:
-                print(t)
+            t.convert_bytestring_to_unicode()
```

Comment:
I think you can put this outside of the loop and just call it once on the table once it's built.